extensions/spotify-player/src/spotify/applescript-scripts.ts

```diff
@@ -0,0 +1,86 @@
+/* eslint-disable no-useless-escape */
+
+export default {
```

Comment:
These scripts should not be too different to the previous versions however I have tried to return as much info as possible and reduce the number of similar but very slightly different functions.